src/topp/SageAdapter.cpp

```diff
@@ -531,7 +531,7 @@ class TOPPSageAdapter :
     for (auto & pid : peptide_identifications) 
     { 
       pid.setIdentifier(identifier);
-      pid.setScoreType("hyperscore");
+      pid.setScoreType("ln(hyperscore)");
```

Comment:
I think the other "hyperscore" we have in OpenMS are also log hyperscores.

Reply:
Then they should be changed as well.

Reply:
Really? This is a breaking change.

Reply:
what we could do: "ln(hyperscore)" for sage and remove X!Tandem in the next version. Not updated for 6+ years